benchmarks/stencil/cuda/stencil_3d_cuda.cu

```diff
@@ -103,13 +103,13 @@ inline void cuda_internal_safe_call(cudaError e, const char* name,
 
 dim3 get_bench_thread_block(size_t y,size_t z) {
   cudaDeviceProp cudaProp;
-  int dim_z = 1;
+  size_t dim_z = 1;
```

Comment:
why did you change?

Reply:
Warning undigned signed in the while loop

Reply:
cast `y` to int instead when you compare